sdcm/cluster_aws.py

```diff
@@ -367,52 +367,95 @@ def network_config_ipv6_workaround_script():  # pylint: disable=invalid-name
+
+            else
+
+                BASE_EC2_NETWORK_URL=http://169.254.169.254/latest/meta-data/network/interfaces/macs/
+                NUMBER_OF_ENI=`curl -s ${BASE_EC2_NETWORK_URL} | wc -w`
+                for mac in `curl -s ${BASE_EC2_NETWORK_URL}`
+                do
+                    DEVICE_NUMBER=`curl -s ${BASE_EC2_NETWORK_URL}${mac}/device-number`
+                    if [[ "$DEVICE_NUMBER" == "1" ]]; then
+                       ETH1_MAC=${mac}
+                    fi
+                done
+                if [[ ! "${DEVICE_NUMBER}x" == "x" ]]; then
+                   ETH1_IP_ADDRESS=`curl -s ${BASE_EC2_NETWORK_URL}${ETH1_MAC}/local-ipv4s`
+                   ETH1_CIDR_BLOCK=`curl -s ${BASE_EC2_NETWORK_URL}${ETH1_MAC}/subnet-ipv4-cidr-block`
                 fi
-            done
+                sudo bash -c "echo 'GATEWAYDEV=eth0' >> /etc/sysconfig/network"
+                echo "
+                DEVICE="eth1"
+                BOOTPROTO="dhcp"
+                ONBOOT="yes"
+                TYPE="Ethernet"
+                USERCTL="yes"
+                PEERDNS="yes"
+                IPV6INIT="no"
+                PERSISTENT_DHCLIENT="1"
+                " > /etc/sysconfig/network-scripts/ifcfg-eth1
+                echo "
+                default via 10.0.0.1 dev eth1 table 2
+                ${ETH1_CIDR_BLOCK} dev eth1 src ${ETH1_IP_ADDRESS} table 2
+                " > /etc/sysconfig/network-scripts/route-eth1
+                echo "
+                from ${ETH1_IP_ADDRESS}/32 table 2
+                " > /etc/sysconfig/network-scripts/rule-eth1
+                sudo systemctl restart network
 
-            if [[ ! "${DEVICE_NUMBER}x" == "x" ]]; then
-               ETH1_IP_ADDRESS=`curl -s ${BASE_EC2_NETWORK_URL}${ETH1_MAC}/local-ipv4s`
-               ETH1_CIDR_BLOCK=`curl -s ${BASE_EC2_NETWORK_URL}${ETH1_MAC}/subnet-ipv4-cidr-block`
             fi
-
-            sudo bash -c "echo 'GATEWAYDEV=eth0' >> /etc/sysconfig/network"
-
-            echo "
-            DEVICE="eth1"
-            BOOTPROTO="dhcp"
-            ONBOOT="yes"
-            TYPE="Ethernet"
-            USERCTL="yes"
-            PEERDNS="yes"
-            IPV6INIT="no"
-            PERSISTENT_DHCLIENT="1"
-            " > /etc/sysconfig/network-scripts/ifcfg-eth1
-
-            echo "
-            default via 10.0.0.1 dev eth1 table 2
-            ${ETH1_CIDR_BLOCK} dev eth1 src ${ETH1_IP_ADDRESS} table 2
-            " > /etc/sysconfig/network-scripts/route-eth1
-
-            echo "
-            from ${ETH1_IP_ADDRESS}/32 table 2
-            " > /etc/sysconfig/network-scripts/rule-eth1
-
-            sudo systemctl restart network
         """)
     # pylint: disable=too-many-arguments
 
+    @staticmethod
+    def installing_ifconfig():
+        return dedent(r"""
+            if `grep -qi "ubuntu" /etc/os-release`; then
+
+                sudo apt update
+                sudo apt install net-tools
+
+            fi
+        """)
+
```

Comment:
May be it is not needed, and you can disable/enable interface with ip command?
`ip link set eht1 down`